cmd/agent/flags.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+)
+
+// flagRunAddr define servers host:port
+// flagReportInterval define time interval in seconds between servers reports
+// flagPollInterval define time interval in seconds between metrics polling
+var (
+	flagRunAddr        string
+	flagReportInterval uint64
+	flagPollInterval   uint64
+)
+
+// Define and parces flags and environment variables
+func parseFlags() {
```

Comment:
```suggestion
func parseFlags() error {
```
Путь Go - сгенерить ошибку и отправить ее наверх (попутно оборачивая). И на самом верху уже принимать решение - что с ней делать. В данном случае на самом верху стоит падать по log.Fatal().